scripts/publish-helm-package.sh

```diff
@@ -18,7 +18,7 @@ rm -rf build
 mkdir build
 cp -r $CHART build
 pushd build
-sed --in-place s/543/$BUILD_NUMBER/ $CHART/Chart.yaml
+sed --in-place s/_ReplaceWithBuildNumber_/$BUILD_NUMBER/ $CHART/Chart.yaml
 helm dependency update --home $HELM_DIR/.helm --verify $CHART
 helm package --home $HELM_DIR/.helm -d $PACKAGE_DIR --version $VERSION $CHART
```

Comment:
Do we need to revert the Chart.yaml change after packaging / push ? Not sure if Jenkins is setup to clone on change, or pull on change into existing workspace.  If the later will Jenkins care if the workspace has uncommitted changes ?